Altis_Life.Altis/core/functions/fn_fetchVehInfo.sqf

```diff
@@ -21,37 +21,28 @@
     11: Max Horse power
     12: Fuel Capacity
 */
-private ["_class","_scope","_picture","_displayName","_vehicleClass","_side","_faction","_superClass","_speed","_armor","_seats","_hp","_fuel"];
-_class = [_this,0,"",[""]] call BIS_fnc_param;
+if !(params [
+    ["_class","",[""]]
+]) exitWith {[]}; //Bad params
```

Comment:
Should be:
```sqf
params [["_class","",[""]]];
```
Next line already exits for bad `params`.

Also, the params return value generally is not very useful, as it reports `false` for at least one or more undefined (optional) arguments. Should be always ignored imo.

Reply:
removed the exit